src/section/shared_state.rs

```diff
@@ -185,10 +184,13 @@ impl SharedState {
         self.our_members.is_adult(name) || self.is_peer_our_elder(name)
     }
 
+    // TODO: review if we still need this function
```

Comment:
This was used to identify nodes that disconnected from us, so we can handle them properly (vote them Offline, etc...). So probably still needed.